relay-event-normalization/src/event.rs

```diff
@@ -109,6 +115,9 @@ pub struct NormalizationConfig<'a> {
     /// advanced normalizations such as inferring contexts or clock drift correction are disabled.
     pub is_renormalize: bool,
 
+    /// Overrides the default flag for other removal.
+    pub remove_other: Option<bool>,
```

Comment:
Maybe in a follow-up we can start removing these optionals?

The `StoreConfig` in the C-ABI can still have the optionals, but the NormalizationConfig doesn't have to have it.

Reply:
Done in [59c0c95](https://github.com/getsentry/relay/pull/3097/commits/59c0c950929b63e64998e8ee817a21e7812b8dba).